src/public/styles.css

```diff
@@ -116,15 +119,19 @@ header {
   align-items: center;
 }
 
+.signup-h2 {
```

Comment:
I'd work slightly on your class names. I wouldn't put h2 in the name personally. I'd probably have something like:
```
<main class="signup">
    <h1 class="signup-title">
```